source/reference/minio-mc/mc-idp-ldap.rst

```diff
@@ -0,0 +1,292 @@
+.. _minio-mc-idp-ldap:
+
+===============
+``mc idp ldap``
+===============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc idp ldap
+
+.. versionadded:: RELEASE.2023-05-26T23-31-54Z
+
+Description
+-----------
+
+.. start-mc-idp-ldap-desc
+
+The :mc-cmd:`mc idp ldap` commands allow you to add, modify, review, list, remove, enable, and disable server configurations to 3rd party :ref:`Active Directory or LDAP Identity and Access Management (IAM) integrations <minio-external-identity-management-ad-ldap>`.
+
+.. end-mc-idp-ldap-desc
+
+Define configuration settings as an alternative to using environment variables when :ref:`setting up an AD/LDAP connection <minio-authenticate-using-ad-ldap-generic>`.
+
+.. note::
+
+   Configuration settings do **not** override settings configured as environment variables.
```

Comment:
I would reverse this and link out. Harder to single source I guess but more useful to the user.

```suggestion
.. note::

   MinIO :ref:`AD/LDAP environment variables <minio-server-envvar-external-identity-management-ad-ldap>` override their corresponding configuration settings as modified or set by this command.
```

Reply:
And apply the same to the openid page

Reply:
Added manually, thank you!